lib/builder.js

```diff
@@ -489,6 +513,13 @@ Builder.prototype.setsAutoHeaders = function() {
   return this.autoHeaders;
 };
 
+/**
+ * Returns true if the given payload will be send without stringifying it first.
```

Comment:
s/send/sent

Reply:
All borked past participles fixed, thanks for playing spell checker for me :-)